src/library_fs.js

```diff
@@ -1110,6 +1110,9 @@ mergeInto(LibraryManager.library, {
       if (!stream.seekable || !stream.stream_ops.llseek) {
         throw new FS.ErrnoError({{{ cDefine('ESPIPE') }}});
       }
+      if (whence < 0 /* SEEK_SET */ || whence > 2 /* SEEK_END */) {
```

Comment:
How about checking if it differs from the 3 possible valid values? (I worry a libc update in the future would not keep those as simple continguous integers, and in that order.)